src/compiler/typescript_to_rust/azle_generate/src/candid_type/func/rust.rs

```diff
@@ -18,7 +18,7 @@ pub fn generate_list_into_vm_value_impl(function_name: String) -> TokenStream {
     quote! {
         impl CdkActTryIntoVmValue<&mut boa_engine::Context, boa_engine::JsValue> for Vec<#function_name> {
             fn try_into_vm_value(self, context: &mut boa_engine::Context) -> Result<boa_engine::JsValue, CdkActTryIntoVmValueError> {
-                try_into_vm_value_generic_array(self, context)
+                _azle_try_into_vm_value_generic_array(self, context)
```

Comment:
We shouldn't prefix our method names, instead we will prefix their methods and export them with the correct name using ic_cdk macros

Reply:
This will be done later. See https://github.com/demergent-labs/cdk_framework/issues/66